### DISCUSSION TITLE
Best VMs for Zinc (RAM  / CPU ratio)

### DISCUSSION THREAD

**Discussion by collimarco** [zinclabs/zinc#277] [General]
Hello,

I am wondering if you can recommend a type of DigitalOcean droplets for Zinc:

- CPU-Optimized: 2GB of RAM for each dedicated vCPU
- General Purpose: 4GB of RAM for each dedicated vCPU
- Memory-Optimized: 8GB of RAM for each dedicated vCPU

And there are also "Basic", which I would discard due to shared CPUs, and "Storage-Optimized", which are like "Memory-Optimized" with faster SSDs.

I wonder **what is the best RAM  / CPU ratio** for Zinc and if it relies more on RAM, CPU or SSD speed.

**Comment by prabhatsharma** [comment]
While the instance type that works well fro your use case  will depend on the kind of traffic that you have. However for most users a CPU optimized instance will work provide best bang for the buck. I am not too familiar with digital ocean droplets, but on AWS you could pick compute optimize instances like c5.large etc. - https://aws.amazon.com/ec2/instance-types/
